lib/drives/pages/drive_detail_page.dart

```diff
@@ -163,6 +164,23 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
         widgets.add(ProfileWrapList(riders, title: S.of(context).riders));
       }
 
+      List<Ride> pendingRides = _drive!.pendingRides!.toList();
+      if (pendingRides.isNotEmpty) {
+        List<Widget> pendingRidesColumn = [
+          const SizedBox(height: 5.0),
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Text(
+              S.of(context).pageDriveChatRequestsHeadline,
+              style: Theme.of(context).textTheme.titleLarge,
+            ),
+          ),
+          const SizedBox(height: 10.0),
+        ];
+        pendingRidesColumn.addAll(_pendingRidesList(pendingRides));
```

Comment:
you don't need `addAll` here, you could put  `... _pendingRidesList(pendingRides)` in the last line of the column array definition

Reply:
thanks for the hint, and sorry that this took two commits.